src/thunderbots/software/CMakeLists.txt

```diff
@@ -233,4 +233,27 @@ if (CATKIN_ENABLE_TESTING)
 
     target_link_libraries(ros_message_util_test ${catkin_LIBRARIES})
 
+    catkin_add_gtest(world_test
+            test/world/world.cpp
+            test/test_util/util.cpp
+            ai/world/world.cpp
+            ai/world/ball.cpp
+            ai/world/field.cpp
+            ai/world/robot.cpp
+            ai/world/team.cpp)
+
+    target_link_libraries(world_test ${catkin_LIBRARIES})
+
+    catkin_add_gtest(unit_test_util_test
```

Comment:
Might just be me, but I think `test_util_test` would be a bit more comprehensible.

Reply:
Makes sense, matches the folder name better. Fixed